hack/test_remotesecrets.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+set -e
+
+RS_NAMESPACE=${1:-default}
+TARGET_NS_1="spi-test-target1"
```

Comment:
make these just defaults so that one can override them from the command line?

I.e. define the vars as `TARGET_NS_1=${TARGET_NS_1:-"spi-test-target"}`.

This could then be called like `RS_NAME=asdf hack/test_remotesecrets.sh`.